pom.xml

```diff
@@ -118,7 +118,6 @@
         <version.org.wildfly.galleon-plugins>4.2.8.Final</version.org.wildfly.galleon-plugins>
         <!-- wildfly plugin-->
         <version.wildfly.plugin>2.0.2.Final</version.wildfly.plugin>
-        <version.org.wildfly.jar.plugin>2.0.0.Beta4</version.org.wildfly.jar.plugin>
```

Comment:
@jfdenise could you please keep this property in `pom.xml` instead of moving it to `testsuite/pom.xml`.

Even though the dependency is only used in the test suite, having the version defined in `pom.xml` make sure we keep a centralized place for all versions of the components we depend on

Reply:
Thank-you. Done.